contracts/PvpArena.sol

```diff
@@ -871,4 +844,49 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
         //this is not final, but processing the loser will update the ranks leaving this player in the 4th position, which will be quickly replaced by other players
         processLoser(characterID);
     }
+
+    /// @dev distributes the ranking rewards pool to top players
+    // TODO: Re-evaluate if private or requires some permission
+    function _distributeSeasonRewards() private {
```

Comment:
No need for permissions if this is private, as long as the function calling ti is properly secured